src/main/java/org/spongepowered/common/mixin/core/server/management/MixinPlayerInstance.java

```diff
@@ -24,15 +24,33 @@
  */
 package org.spongepowered.common.mixin.core.server.management;
 
+import net.minecraft.network.Packet;
+import net.minecraft.network.play.server.S21PacketChunkData;
+import net.minecraft.server.management.PlayerManager;
 import net.minecraft.server.management.PlayerManager.PlayerInstance;
+import net.minecraft.world.ChunkCoordIntPair;
+import net.minecraft.world.chunk.Chunk;
 import net.minecraft.world.gen.ChunkProviderServer;
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.common.interfaces.IMixinChunk;
+import org.spongepowered.common.interfaces.server.management.IMixinPlayerInstance;
 import org.spongepowered.common.interfaces.world.IMixinWorldServer;
 
 @Mixin(PlayerInstance.class)
-public class MixinPlayerInstance {
+public abstract class MixinPlayerInstance implements IMixinPlayerInstance {
+
+    @Shadow(aliases = "field_73265_a") private PlayerManager this$0;
```

Comment:
It would be easier to read if you give this a proper name and add `this$0` to the aliases instead